src/Services/QueuedJobService.php

```diff
@@ -66,6 +67,29 @@ class QueuedJobService
      */
     private static $stall_threshold = 3;
 
+    /**
+     * How early stuck jobs will become eligible for automated retry processing (minutes)
+     */
+    private static int $job_retry_buffer = 1;
+
+    /**
+     * How many jobs can be automatically retried per health check
+     * Setting this to 0 disables the job retry feature
+     */
+    private static int $job_retry_limit = 10;
+
+    /**
+     * Map of stuck job statuses to new status when retrying the job.
+     * For example a "broken" job should be turned into a "new" job for it to be retried correctly.
+     * Set the value to "null" for any given stuck job status to disable retries for that status.
```

Comment:
```suggestion
     * Set the value to `null` for any given stuck job status to disable retries for that status.
```
Sorry, I know I had the quote marks in my suggestion but reading this now it makes it seem like it should be set to the string `"null"` not the value `null`.